README.md

```diff
@@ -93,19 +93,26 @@ An intf uses the primitives defined in pkgs to describe the interface between tw
 An intf consists of one or more components, and each component consists of one or more connections.  
 
 components are groups of individual port-to-port connections that constitute a higher-level protocol or logical grouping of signals.
-For example, if two blocks had a val/rdy handshake, the "val" port and the "rdy" port would each be described in a connection, and those connections would be contained within a component.
+Ports describe how an interface (component) looks like in terms of width and direction.
```

Comment:
Why is 'component' used in parenthesis throughout this README? I think you can describe what a component is relative to an interface once near the top of the `yis_intf` section, then reference 'component' later on in the docs without spelling out the full 'interface (component)'

Reply:
updated